web/annotation_layer_builder.css

```diff
@@ -239,50 +243,58 @@
   appearance: none;
 }
 
-.annotationLayer .popupTriggerArea {
-  height: 100%;
-  width: 100%;
+.annotationLayer .fileAttachmentAnnotation canvas {
+  pointer-events: none;
```

Comment:
Do we actually need this rule now, given that you added [this line yesterday](https://github.com/mozilla/pdf.js/blob/bd46a934ece88f3ba6580a6ba89c801bb2656c0f/web/annotation_layer_builder.css#L70)?